firefox_puppeteer/__init__.py

```diff
@@ -47,6 +47,14 @@ def keys(self):
         See the :class:`~api.keys.Keys` reference.
         """
 
+    @use_class_as_property('api.permissions.Permissions')
+    def perms(self):
+        """
+        Provides an api for interacting with permissions.
+
+        See the :class:`~api.permissions.Permissions` reference.
```

Comment:
Hi, @whimboo!

I wonder whether we want to maintain a small permissions library, or should we add this to the utils library created in PR #100?

For whatever it's worth here, here's another utils.js function that we need for this PR, utils.assertLoadedUrlEqual, which enables us to check that URLs are equal after a redirect.